2018-04-23 09:55:14.130 [ERROR] [vert.x-eventloop-thread-0] com.zaxxer.hikari.pool.HikariPool [567] - HikariPool-1 - Exception during pool initialization.
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_161]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_161]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_161]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_161]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:981) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:339) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2253) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44) ~[Database-Connect-1.0.jar:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_161]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_161]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_161]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_161]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.jdbc2.optional.MysqlDataSource.getConnection(MysqlDataSource.java:422) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.jdbc2.optional.MysqlDataSource.getConnection(MysqlDataSource.java:134) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:365) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:194) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:460) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:534) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81) ~[Database-Connect-1.0.jar:?]
	at obj.SqlConnector.initPool(SqlConnector.java:82) ~[Database-Connect-1.0.jar:?]
	at obj.SqlConnector.<init>(SqlConnector.java:29) ~[Database-Connect-1.0.jar:?]
	at app.ModuleService.installJooq(ModuleService.java:89) ~[Database-Connect-1.0.jar:?]
	at app.ModuleService.configure(ModuleService.java:81) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.AbstractModule.configure(AbstractModule.java:62) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.spi.Elements$RecordingBinder.install(Elements.java:340) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.spi.Elements.getElements(Elements.java:110) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.internal.InjectorShell$Builder.build(InjectorShell.java:138) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:104) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.Guice.createInjector(Guice.java:96) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.Guice.createInjector(Guice.java:73) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.Guice.createInjector(Guice.java:62) ~[Database-Connect-1.0.jar:?]
	at app.Start.start(Start.java:18) ~[Database-Connect-1.0.jar:?]
	at io.vertx.core.AbstractVerticle.start(AbstractVerticle.java:111) ~[Database-Connect-1.0.jar:?]
	at io.vertx.core.impl.DeploymentManager.lambda$doDeploy$10(DeploymentManager.java:481) ~[Database-Connect-1.0.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$wrapTask$2(ContextImpl.java:344) ~[Database-Connect-1.0.jar:?]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [Database-Connect-1.0.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:403) [Database-Connect-1.0.jar:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:463) [Database-Connect-1.0.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858) [Database-Connect-1.0.jar:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method) ~[?:1.8.0_161]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[?:1.8.0_161]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[?:1.8.0_161]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[?:1.8.0_161]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[?:1.8.0_161]
	at java.net.Socket.connect(Socket.java:589) ~[?:1.8.0_161]
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:211) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:298) ~[Database-Connect-1.0.jar:?]
	... 41 more
2018-04-23 09:58:14.076 [ERROR] [vert.x-eventloop-thread-0] com.zaxxer.hikari.pool.HikariPool [567] - HikariPool-1 - Exception during pool initialization.
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_161]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_161]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_161]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_161]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:981) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:628) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1014) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44) ~[Database-Connect-1.0.jar:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_161]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_161]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_161]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_161]
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.jdbc2.optional.MysqlDataSource.getConnection(MysqlDataSource.java:422) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.jdbc2.optional.MysqlDataSource.getConnection(MysqlDataSource.java:134) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:365) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:194) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:460) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:534) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115) ~[Database-Connect-1.0.jar:?]
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81) ~[Database-Connect-1.0.jar:?]
	at obj.SqlConnector.initPool(SqlConnector.java:82) ~[Database-Connect-1.0.jar:?]
	at obj.SqlConnector.<init>(SqlConnector.java:29) ~[Database-Connect-1.0.jar:?]
	at app.ModuleService.installJooq(ModuleService.java:89) ~[Database-Connect-1.0.jar:?]
	at app.ModuleService.configure(ModuleService.java:81) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.AbstractModule.configure(AbstractModule.java:62) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.spi.Elements$RecordingBinder.install(Elements.java:340) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.spi.Elements.getElements(Elements.java:110) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.internal.InjectorShell$Builder.build(InjectorShell.java:138) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:104) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.Guice.createInjector(Guice.java:96) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.Guice.createInjector(Guice.java:73) ~[Database-Connect-1.0.jar:?]
	at com.google.inject.Guice.createInjector(Guice.java:62) ~[Database-Connect-1.0.jar:?]
	at app.Start.start(Start.java:18) ~[Database-Connect-1.0.jar:?]
	at io.vertx.core.AbstractVerticle.start(AbstractVerticle.java:111) ~[Database-Connect-1.0.jar:?]
	at io.vertx.core.impl.DeploymentManager.lambda$doDeploy$10(DeploymentManager.java:481) ~[Database-Connect-1.0.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$wrapTask$2(ContextImpl.java:344) ~[Database-Connect-1.0.jar:?]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163) [Database-Connect-1.0.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:403) [Database-Connect-1.0.jar:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:463) [Database-Connect-1.0.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858) [Database-Connect-1.0.jar:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]
Caused by: java.io.EOFException: Can not read response from server. Expected to read 4 bytes, read 0 bytes before connection was unexpectedly lost.
	at com.mysql.jdbc.MysqlIO.readFully(MysqlIO.java:2957) ~[Database-Connect-1.0.jar:?]
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:560) ~[Database-Connect-1.0.jar:?]
	... 42 more
2018-04-23 14:48:13.774 [ERROR] [vert.x-worker-thread-0] app.ServiceImpl [75] - null
